src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -96,6 +98,17 @@
   private static final String RECALL_REQUEST_EVENT = "Recall request";
   private static final String ITEM_RECALLED = "Item recalled";
 
+  @Before
+  public void setupLoanType() {
+    noteTypeFixture.generalNoteType();
+  }
```

Comment:
The name refers to a loan type yet the implementation refers to a note type, could this be renamed to remove this discrepancy?